journal/journal_unix.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !windows
```

Comment:
Uhm, this file is already called `_unix.go` so I think this (and the existing annotation just below too) may not be needed. Did you see something which suggests otherwise?

Reply:
Ah bummer, I didn't realized `unix` and `linux` behave differently in regard to this. Thanks for the reference!